UOP1_Project/Assets/Scripts/Sound/SoundManager.cs

```diff
@@ -0,0 +1,163 @@
+﻿using System.Collections;
+using System.Linq;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Audio;
+
+public class SoundManager : MonoBehaviour
+{
+	public static SoundManager singleton;
```

Comment:
So I have one nitpick and one suggestion that probably needs to happen:

Nitpick: Conventionally, the singleton pattern calls its static instance `Instance`. At the very least, it should be `Singleton`, since it's a public field.
Suggestion:
 ```suggestion
       public static SoundManager Instance;
         
       [RuntimeInitializeOnLoadMethod(RuntimeInitializeLoadType.SubsystemRegistration)]
       static void Init()
       {
               Instance = default;
       }
```
This makes the static variable compatible with Fast Enter Play Mode, which is a requirement in the conventions doc.

Reply:
I was under the impression that [RuntimeInitializeOnLoadMethod] was inconsistent and never used it, I'll take a look at it today and probably accept the commit.